security/Ch-APIGateway.rst

```diff
@@ -13,7 +13,7 @@ KONG (https://konghq.com/) is the product underlying the API gateway.  The EdgeX
 Start the API Gateway
 ======================
 
-Start the API gateway with Docker Compose and a Docker Compose manifest file (the Docker Compose file named docker-compose-edinburgh-1.0.0.yml found at `https://github.com/edgexfoundry/developer-scripts/blob/master/compose-files/docker-compose-edinburgh-1.0.0.yml <https://github.com/edgexfoundry/developer-scripts/blob/master/compose-files/docker-compose-edinburgh-1.0.0.yml>`_). Save it as ``docker-compose.yml``.  This Compose file starts all of EdgeX including the security services. The command to start EdgeX inclusive of API gateway related services is:
+Start the API gateway with Docker Compose and a Docker Compose manifest file (the Docker Compose file named docker-compose-edinburgh-1.0.1.yml found at https://github.com/edgexfoundry/developer-scripts/tree/master/releases/edinburgh/compose-files/security)).  This Compose file starts all of EdgeX including the security services. The command to start EdgeX inclusive of API gateway related services is:
```

Comment:
https://github.com/edgexfoundry/developer-scripts/tree/master/releases/edinburgh/compose-files/security doesn't exist.  Security version(s) of docker-compose files are in the same directory as the non-security version(s) for a given release.

Reply:
Fixed